### PR TITLE
Specify firsttechfed.com custom-software implementation

### PR DESCRIPTION
**Pull request by stephenjust**
First Tech Fed uses Entrust Identity Guard as their solution for soft tokens.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Would you happen to know the name of their hardware keys as well?

The [documentation](https://www.firsttechfed.com/help/security/prevent#browse) page is hilarious to read! Not only do they advice people to look for the deprecated EV cert browser text but they also don't even use EV certs! It's a bit hard to verify the EV text on a Lets Encrypt certificate :laughing:

**Comment by stephenjust** [comment]
Entrust is their vendor for hardware tokens as well - or at least that's what their customer service suggested. I'm much less confident in this though so I didn't include it.

File: entries/f/firsttechfed.com.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
       "call",
       "email"
     ],
+    "custom-software": [
+      "Entrust Identity Guard"
+    ],
```
Comment (Carlgo11) [inline entries/f/firsttechfed.com.json:14]: After reviewing the data you've provided and what was previously provided at #5516 I think `Entrust Hardware Token` can be added as custom-hardware.
The previous submitter also mentioned sms, phone calls and email 2FA. Is that still something that is provided?

```suggestion
    ],
    "custom-hardware": [
      "Entrust Hardware Token"
    ],
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/f/firsttechfed.com.json (hunk 1 above)
Comment (stephenjust) [reply entries/f/firsttechfed.com.json:14]: Yes, the other data is still correct.

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**